### PR TITLE
Shamim back button

### PR DESCRIPTION
**Pull request by Sham199**
-Added CSS for assign to project page 

-Edited CSS and class names to follow naming convention (e g replaced container4 with container3 etc)

-Added back buttons to assign to project and project details page 

-Edited back button link in upload impact record 

-Edited upload more research files page layout

-Reviewed the Meeting minutes for formatting issues and separated them into 2 parts for the first and second submission

-Added Acceptance testing and Code inspection report to the Testing Document & uploaded the most recent version of Testing document

### CODE REVIEW

File: Software/pages/adminhome.php
Comment (jorubka) [inline Software/pages/adminhome.php:1]: this page needs to be removed

File: Software/pages/impactproject.php
Comment (jorubka) [inline Software/pages/impactproject.php:1]: page to be deleted

File: Software/pages/projectDetails.php
Comment (jorubka) [inline Software/pages/projectDetails.php:1]: no need to add content of header twice
<title> can be added after <?php require header.php?>

File: Software/pages/projects.php
Comment (jorubka) [inline Software/pages/projects.php:1]: no need to add content of header twice

File: Software/pages/resimpproject.php
Comment (jorubka) [inline Software/pages/resimpproject.php:1]: page to be deleted.

File: Software/pages/uploadGrants.php
Comment (jorubka) [inline Software/pages/uploadGrants.php:1]: no need to add content of header twice

File: Software/pages/uploadMoreResearchFiles.php
Comment (jorubka) [inline Software/pages/uploadMoreResearchFiles.php:1]: no need to add content of header twice
title after header.php

File: Software/pages/uploadimpact.php
Comment (jorubka) [inline Software/pages/uploadimpact.php:1]: no need to add content of header twice

File: Software/pages/uploadresearch.php
Comment (jorubka) [inline Software/pages/uploadresearch.php:1]: no need to add content of header twice

File: Software/pages/userProfile.php
Comment (jorubka) [inline Software/pages/userProfile.php:1]: no need to add content of header twice

File: Software/pages/users.php
Comment (jorubka) [inline Software/pages/users.php:1]: in head section only <?php require header.php?> and <title>

File: Software/pages/viewAllImpact.php
Comment (jorubka) [inline Software/pages/viewAllImpact.php:1]: no need to add content of header twice

File: Software/php/displayProjectDetails.php
Comment (jorubka) [inline Software/php/displayProjectDetails.php:1]: download zip option must be on the top of the page; 

needs adding that part on the beginning of the page. 

require '../db/dbconnect.php'; 
 if ($_SERVER['REQUEST_METHOD']== 'POST') {
        $projectID = $_POST['projectID'];
   }

**Comment by jorubka** [comment]
needs correction, we need to review this pull request before pushing

File: Software/pages/home.php
[the remark this answers is not in the document]
Comment (Sham199) [reply Software/pages/home.php:1]: Done

File: Software/pages/assignToProject.php
[the remark this answers is not in the document]
Comment (Sham199) [reply Software/pages/assignToProject.php:1]: Done

File: Software/pages/projectDetails.php
Comment (Sham199) [reply Software/pages/projectDetails.php:1]: Done

File: Software/pages/uploadGrants.php
Comment (Sham199) [reply Software/pages/uploadGrants.php:1]: Done

File: Software/pages/uploadMoreResearchFiles.php
Comment (Sham199) [reply Software/pages/uploadMoreResearchFiles.php:1]: Done

File: Software/pages/uploadimpact.php
Comment (Sham199) [reply Software/pages/uploadimpact.php:1]: Done

File: Software/pages/uploadresearch.php
Comment (Sham199) [reply Software/pages/uploadresearch.php:1]: Done

File: Software/pages/users.php
Comment (Sham199) [reply Software/pages/users.php:1]: Done

File: Software/pages/viewAllImpact.php
Comment (Sham199) [reply Software/pages/viewAllImpact.php:1]: Done

File: Software/pages/viewImpactProjects.php
[the remark this answers is not in the document]
Comment (Sham199) [reply Software/pages/viewImpactProjects.php:1]: Done

File: Software/pages/adminhome.php
Comment (Sham199) [reply Software/pages/adminhome.php:1]: Done

File: Software/pages/impactproject.php
Comment (Sham199) [reply Software/pages/impactproject.php:1]: Done

File: Software/pages/resimpproject.php
Comment (Sham199) [reply Software/pages/resimpproject.php:1]: Done

**Comment by jorubka** [comment]
All looks good now

**Pull request closed without merging**